src/components/data/toolbar/DataDotMenu.js

```diff
@@ -88,7 +90,7 @@ function DataDotMenu(props) {
         : null;
     const deleteMiEnabled = !isSelectionEmpty && isOwner(selectedResources);
     const metadataMiEnabled = selected?.length === 1;
-
+    const requestDoiEnabled = isOwner(selectedResources);
```

Comment:
I could be wrong, not the expert on this, but I think DOI requests also have to be done on folders and not files?

Reply:
Yes, this should also ensure that only 1 folder selected.